packages/riipen-ui/src/components/Paper.jsx

```diff
@@ -0,0 +1,111 @@
+import clsx from "clsx";
+import PropTypes from "prop-types";
+import React from "react";
+import _JSXStyle from "styled-jsx/style";
+
+import ThemeContext from "../styles/ThemeContext";
+import withClasses from "../utils/withClasses";
+
+const Paper = props => {
+  const {
+    children,
+    classes,
+    component: Component,
+    rounded,
+    elevation,
+    variant,
+    ...other
+  } = props;
+
+  const theme = React.useContext(ThemeContext);
+
+  const className=clsx(
+    "root",
+    classes,
+    rounded ? "rounded" : null,
+    variant === "outlined" ? "outlined" : "elevation"
+  );
+  
+  return (
+    <React.Fragment>
+    <Component 
+    className={className}
+      {...other}
+    >
+      {children}
+      </Component>
+     
+      <style jsx>
+        {`
+          .root {
+            background-color: ${theme.palette.common.white};
+            color: ${theme.palette.text.primary}
+            padding: ${theme.spacing(3)}px;
+            z-index: ${theme.zIndex.low};
```

Comment:
I'd remove this. `z-index` can be a pretty dangerous thing to play with. A good rule of thumb is if you don't absolutely need to set, just leave it as whatever default the browser gives it.

Reply:
Should the background color stay in there though?